features_and_tools/restful_api.rst

```diff
@@ -23,6 +23,10 @@ Firely Server enables create-on-update: If you request an update and no resource
 
 Firely Server can reject a resource based on :ref:`feature_prevalidation`.
 
+Update with no changes
```

Comment:
I would create a separate header with all the information on the 'UpdateNoOp' plugin combined, rather than divide the information among separate headers. This way you can also create an anchor in the rst, such as:

.. _UpdateNoOp:

making it easier to link to this information.

Reply:
separate header is added